src/main/resources/templates/errors/error-403.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>403 Error</title>
+</head>
+<body>
+
+<h1 th:text="${errorTitle}"></h1>
+<hr>
+
+<th:block th:if="${errorMessage == null}">
+    <h3>You're not authorized!</h3>
```

Comment:
<h3>You're not authorized!</h3> this is a generic message for Status code 401 (not authorized) and must be adapted to 403 (forbidden). 
**[Here is a list with status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)**